lib/src/file_watcher/polling.dart

```diff
@@ -37,13 +37,13 @@ class _PollingFileWatcher implements FileWatcher, ManuallyClosedWatcher {
   final _readyCompleter = Completer();
 
   /// The timer that controls polling.
-  Timer _timer;
+  late Timer _timer;
```

Comment:
Can we make this `late final` and move the initializing expression.

Reply:
I will make it `final`, but `_timer = Timer.periodic(pollingDelay, (_) => _poll());` requires `pollingDelay` parameter, which is not a field yet. So, I keep assigning in the constructor.